app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controllers/topics-controller');
+//const { handleServerErrors } = require('./controllers/error-handling-controllers')
```

Comment:
get this error handler back in there, if you are invoking next with the error in the catch blocks of your controllers then that should be what tells express to use the error handler :)